avogadro/qtplugins/svg/CMakeLists.txt

```diff
@@ -0,0 +1,15 @@
+include_directories(${CMAKE_CURRENT_BINARY_DIR})
+find_package(Qt5 COMPONENTS Svg REQUIRED)
+
+
+avogadro_plugin(SVG
+  "Project the screen in a SVG image."
+  ExtensionPlugin
+  svg.h
+  SVG
+  "svg.cpp"
+  ""
+)
+
+target_link_libraries(SVG LINK_PRIVATE Qt5::Svg)
```

Comment:
Should combine into a single line

Reply:
I think pretty much *all* the plugins have been using multiple lines here, so let's work on that in a separate change.